### DISCUSSION TITLE
Inclusion in LibraZiK-4

### DISCUSSION THREAD

**Discussion by trebmuh** [hannesbraun/airwindows-lv2#12] [Show and tell]
Hi Hannes and all

Just to let you know that as of today, AirWindows-lv2 14.0 is accessible for all the [LibraZiK-4](https://librazik.tuxfamily.org/base-site-LZK/english.php) users.
Thanks for the work you've done so far!

_Edit: I've made a [Mastodon post](https://mastodon.xyz/@LibraZiK/109621084679267614) for this._

**Comment by hannesbraun** [comment]
Neat! That's great to hear.

*I definitely need to put in more work into this project again as soon as there's time... 😅*

**Reply by trebmuh** [reply]
As we just started a new year, I'd say that there is plenty of time available upfront :smile: 
Anyway, thank you very much for those >80 plugins already ported to LV2, that's a great effort!
